project/Dependencies.scala

```diff
@@ -324,6 +324,16 @@ object Dependencies {
       )
   )
 
+  val JakartaJms = Seq(
+    libraryDependencies ++= Seq(
+        "jakarta.jms" % "jakarta.jms-api" % "3.1.0", // Eclipse Public License 2.0 + + GPLv2
+        "org.apache.activemq" % "artemis-jakarta-server" % "2.31.2" % Test, // ApacheV2
+        "org.apache.activemq" % "artemis-jakarta-client" % "2.31.2" % Test, // ApacheV2
```

Comment:
Had to rewrite the tests to use Artemis instead. Was not trivial, but working now. 139f1f2